### PR TITLE
Remove Cancel button, one (Back) is enough

### PR DESCRIPTION
**Pull request by mathjazz**
@jotes @Osmose r?

### CODE REVIEW

File: pontoon/base/static/js/translate.js [hunk 1]
```diff
@@ -1842,13 +1853,16 @@ var Pontoon = (function (my) {
...
           break;
 
         case "UPDATE":
-          var entity = Pontoon.entities[message.value.id];
-          Pontoon.updateOnServer(entity, message.value.content, true, true);
+          var entity = Pontoon.entities[message.value.id],
+              translation = message.value.content;
+          Pontoon.updateOnServer(entity, translation, true, true);
+          $('#translation').val(translation);
+          Pontoon.updateCachedTranslation();
```
Comment (mathjazz) [inline pontoon/base/static/js/translate.js]: This is a bugfix I noticed while working on this improvement. It caused unsaved changes warning to appear if translation was submitted inplace followed by openning another entity.

File: pontoon/base/static/js/translate.js [hunk 2]
```diff
@@ -717,7 +745,10 @@ var Pontoon = (function (my) {
           if ($('.warning-overlay').is(':visible')) {
             $('.warning-overlay .cancel').click();
           } else if (!self.app.advanced) {
-            $('#cancel').click();
+            self.checkUnsavedChanges(function() {
+              self.goBackToEntityList();
+              self.stoplInPlaceEditing();
```
Comment (jotes) [inline pontoon/base/static/js/translate.js]: stoplIn? did you meant stopIn?

Comment (mathjazz) [inline pontoon/base/static/js/translate.js]: Huh, nice catch. ;)

**Comment by Osmose** [comment]
r+, nice work!

**Pull request merged**